central/src/components/GameDashboard.jsx

```diff
@@ -88,7 +88,7 @@ export default function GameDashboard({ loading, games, deleteGame, cloneGame, g
       return games
         .map((game, index) => {
           const { id, title, description, cluster, domain, grade, standard, imageUrl } = game;
-          const questionCount = game?.questions?.length || 0;
+          const questionCount = game?.questionTemplates?.length || 0;
```

Comment:
Instead of having `questionTemplates` to be nullable, should we just use an empty array when parsing server's response for it?